hack/aws-nuke-config.yaml

```diff
@@ -0,0 +1,96 @@
+regions:
+  - us-east-2
```

Comment:
maybe we can extract this from this file and also the workflow file. I think this might cause subtle problems if we duplicate it.